app/assets/javascripts/search.js

```diff
@@ -23,4 +23,25 @@ jQuery(document).ready(function() {
     }
   })
 
+  $('#questions_searchform').submit(function(e){ 
+    e.preventDefault()
+    window.location = '/search/questions/'+$('#questions_searchform_input').val()
```

Comment:
@ujithaperera Sure Ujitha feel free to make any changes in the questions search option. To provide the underline it should search any questions related to the keyword and if not found should redirect to the post url.